modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @file
+ * The iNope Policy Review Module.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function inope_policy_review_menu() {
+  $items = array();
+
+  $items['node/%/review'] = array(
+    'title callback' => 'inope_policy_review_tab_title_callback',
+    'title arguments' => array(1),
+    'description' => 'Review of policy nodes.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('inope_policy_review_form'),
+    'access arguments' => array('view published content'),
+    'type' => MENU_LOCAL_TASK,
+  );
+
+  return $items;
+
+}
+
+/**
+ * The iNope Policy Review Page Title Callback.
+ */
+function inope_policy_review_tab_title_callback($node_nid) {
+  $node = node_load($node_nid);
+  $node_title = $node->title;
+  $title = t('Review of @node_title', array('@node_title' => $node_title));
+  return $title;
+}
+
+/**
+ * The iNope Policy Review Form.
+ */
+function inope_policy_review_form($form, &$form_state) {
+
+  $form['inope_policy_review_approved_checkbox'] = array(
```

Comment:
Check this out:

http://www.nngroup.com/articles/checkboxes-vs-radio-buttons/

Idk about the quality of the article, honestly I think I learned form controls from visual basic when you were probably still in grade school, so I can't remember my trajectory to learn this stuff.  Anyway the important part is: 

```
    Radio buttons are used when there is a list of two or more options that are mutually exclusive and the user must select exactly one choice. In other words, clicking a non-selected radio button will deselect whatever other button was previously selected in the list.
    Checkboxes are used when there are lists of options and the user may select any number of choices, including zero, one, or several. In other words, each checkbox is independent of all other checkboxes in the list, so checking one box doesn't uncheck the others.
    A stand-alone checkbox is used for a single option that the user can turn on or off.
```

You will probably want to review HTML form controls -- you want to know when to use one vs the other, it does make a UX difference. If you need help finding some resources I can see what I can do -- but take a whack at finding some out there.